packages/migrate/src/__tests__/DbPull.test.ts

```diff
@@ -485,11 +501,300 @@ describe('postgresql', () => {
     expect(ctx.mocked['process.stdout.write'].mock.calls.join('\n')).toMatchInlineSnapshot(`
 
 
-            - Introspecting based on datasource defined in prisma/using-dotenv.prisma
+                              - Introspecting based on datasource defined in prisma/using-dotenv.prisma
 
-            ✖ Introspecting based on datasource defined in prisma/using-dotenv.prisma
+                              ✖ Introspecting based on datasource defined in prisma/using-dotenv.prisma
 
-        `)
+                                                    `)
+    expect(ctx.mocked['process.stderr.write'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
+  })
+})
+
+describeIf(!process.env.TEST_SKIP_MSSQL)('sqlserver-multi-schema', () => {
+  if (process.env.CI) {
+    // to avoid timeouts on macOS
+    jest.setTimeout(80_000)
+  } else {
+    jest.setTimeout(20_000)
+  }
```

Comment:
How about setting a single `jest.setTimeout(80_000)`?

Reply:
It's a copy paste from the other sql server describe and it works so I don't think we need to change it really